azure-pipelines.yml

```diff
@@ -148,7 +148,7 @@ stages:
       submodules: True
     - script: |
         brew install flex bison cmake python@3
-        python3 -m pip install --upgrade pip setuptools
+        python3 -m pip install --upgrade pip "setuptools<=70.3.0"
```

Comment:
Where does this requirement come from? (I think I've seen it in the NEURON world.)

Reply:
Otherwise, I get CI failure when installing neuron

Reply:
Alright, if that's what it takes to make it run :)